client/src/App.js

```diff
@@ -1,52 +1,132 @@
-import React from "react";
+import React, {Component} from "react";
 import "./App.css";
 import Search from "./components/Search";
+import Timetable from "./components/TimeTable";
+
+
+class App extends Component {
+  constructor(props)
+  {
+    super(props);
+    this.state = {
+<<<<<<< HEAD
+      isRoot: 1,
+      isPreview: 0
+=======
+      isRoot: true,
+      isPreview: false
+>>>>>>> feature
```

Comment:
You committed a merge conflict here

Reply:
what... :o I'll fix it
It didn't throw any error while running